app/app.go

```diff
@@ -388,7 +402,9 @@ func NewOsmosisApp(
 		ibchost.ModuleName,
 		gammtypes.ModuleName,
 		txfeestypes.ModuleName,
-		genutiltypes.ModuleName, evidencetypes.ModuleName, ibctransfertypes.ModuleName,
+		genutiltypes.ModuleName, evidencetypes.ModuleName, authz.ModuleName,
+		paramstypes.ModuleName, upgradetypes.ModuleName, vestingtypes.ModuleName,
```

Comment:
Are these three all new, or were there bugs before with them not being present?

Reply:
Same as https://github.com/osmosis-labs/osmosis/pull/782#discussion_r793414110

Reply:
one of the things that I did in the annotation branch is "flatten" the *.ModuleName to make it easier to read.

Reply:
@faddat wdym by flatten?